lib/venue.rb

```diff
@@ -1,2 +1,35 @@
 class Venue
+  attr_reader :name,
+              :capacity,
+              :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(patron_name)
+    @patrons << patron_name
+  end
+
+  def yell_at_patrons
+    @patrons.map do |patron|
+      patron.upcase
+    end
+  end
+
+  def over_capacity?
+    if @patrons.length <= @capacity
+      false
+    elsif @patrons.length > @capacity
+      true
+    end
```

Comment:
Whenever you have an `if/else` statement that returns true or false, consider using the condition itself as the return value.